src/assets/server/rest/v1/schemas/chargingstation/chargingstation.json

```diff
@@ -91,6 +91,11 @@
       "type": "number",
       "sanitize": "mongo"
     },
+    "connectorIDs": {
+      "type": "string",
```

Comment:
Isn't is an array on Numbers?

Reply:
This is the user side input, so we receive a pipe separated string here, splited and converted on service side